vm/src/compile.rs

```diff
@@ -1125,6 +1168,14 @@ impl Compiler {
                 self.compile_expression(body)?;
                 self.emit(Instruction::ReturnValue);
                 let code = self.pop_code_object();
+
+                // function doc
+                self.emit(Instruction::LoadConst {
+                    value: bytecode::Constant::String {
+                        value: "".to_string(),
```

Comment:
This empty doc string can be removed if you implement StoreAttr at the functiondef compilation time.